laravel/framework/2020-08-06-1.yaml

```diff
@@ -0,0 +1,14 @@
+title:     Guard bypass in Eloquent models
+link:      https://blog.laravel.com/security-release-laravel-61834-7232
+cve:       ~
+branches:
+    5.5:
+        time:     ~
+        versions: ['>=5.5.0', '<=5.5.49']
```

Comment:
I would put `<60` if this is unpatch, to make it clearer

Reply:
Are you sure? Laravel did not use semver until v6, so v5.5 is actually a major version.